pkg/ocm/placementbinding.go

```diff
@@ -25,6 +25,48 @@ type PlacementBindingBuilder struct {
 	errorMsg string
 }
 
+// NewPlacementBindingBuilder creates a new instance of PlacementBindingBuilder.
+func NewPlacementBindingBuilder(
```

Comment:
Are all of these param required for basic resource?

Reply:
Yes, the kubebuilder validation comments require both of them with all their fields also required